.bazelci/presubmit.yml

```diff
@@ -4,11 +4,23 @@ tasks:
   ubuntu1604:
     name: ":linux:"
     shell_commands:
-    - wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
-    - bash ./Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/miniconda
-    build_targets:
-    - "//plaidml/keras:wheel"
-    build_flags:
+    - curl -O https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh 
+    - sh ./Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/miniconda
+    - mkdir neo
+    - cd neo
```

Comment:
I think we need the CPU drivers, not the NEO (GPU) ones.

Reply:
We've disabled the opencl tests in linux, but I'll keep this in mind if we decide to reenable them.